src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,18 @@
+package duke.task;
+
+/**
+ * Use to represent functions required for a type of task.
+ */
+public interface Task {
+    String getStatusIcon();
+
+    void markAsDone();
+
+    void markAsIncomplete();
```

Comment:
If all the classes that extends Task operate markAsDone() and markAsIncomplete() the same way, can just make Task a class and have that implementation, instead of overriding at every other task type that extends it